src/components/PopupWithForm.js

```diff
@@ -16,6 +18,14 @@ export default class PopupWithForm extends Popup {
     return inputValue;
   }
 
+  setLoadingState(isLoading) {
+    if (this._submitButton) {
+      this._submitButton.textContent = isLoading
```

Comment:
If `isLoading`, then you should add `"Saving..."`, when not loading - the default text